biomolecules_manager.py

```diff
@@ -0,0 +1,15 @@
+import src.dna_rna_tools as dna_rna
+import src.fastq_tools as fastq
+import src.protein_tools as protein
+
+
+def dna_rna_tools(*args):
+    dna_rna.run_dna_rna_tools(*args)
+
+
+def fastq_tools(seqs: set, gc_bounds=(0, 101), length_bounds=(0, 2 ** 32), quality_thresold=0):
+    fastq.fastq_tools(seqs, gc_bounds=gc_bounds, length_bounds=length_bounds, quality_thresold=quality_thresold)
+
+
+def protein_tools(function: str, *prots: str):
+    protein.protein_tools(function, *prots)

```

Comment:
Смотри, все супер, отличные вызовы функций, все корректно импортировано

НО - этот скрипт не будет работать, как ожидается, потому что ни в одной из функций не написан return, все три функции ничего не возвращают

Два простых теста:

`fastq_tools(EXAMPLE)` # пусто
`fastq.fastq_tools(EXAMPLE)` # а вот тут уже получаем верный ответ

И так будет с каждой из 3 высокоуровневых функций, увы

Решается return-ом (предлагаю код, где пофиксила для одной из 3 функций, остальные так же)
```suggestion
def dna_rna_tools(*args):
    return dna_rna.run_dna_rna_tools(*args)


def fastq_tools(seqs: set, gc_bounds=(0, 101), length_bounds=(0, 2 ** 32), quality_thresold=0):
    fastq.fastq_tools(seqs, gc_bounds=gc_bounds, length_bounds=length_bounds, quality_thresold=quality_thresold)


def protein_tools(function: str, *prots: str):
    protein.protein_tools(function, *prots)

```

Кстати... а где аннотация типов для dna_rna_tools и докстринги у всех 3 функций?...